web/src/stream_edit_subscribers.ts

```diff
@@ -466,8 +466,8 @@ export function initialize(): void {
     });
 
     $("#channels_overlay_container").on(
-        "submit",
-        ".edit_subscribers_for_stream .subscriber_list_remove form",
```

Comment:
Switched from using form submission to button click handling, as the form submission served no extra purpose. Does that sound good?